src/lib/decodeData.test.ts

```diff
@@ -38,17 +38,31 @@ describe('decodeData', () => {
     {
       name: 'MyKeyName:<bytes32>:<bool>',
       key: '0x',
-      keyType: 'Singleton',
+      keyType: 'MappingWithGrouping',
       valueType: 'bytes',
       valueContent: 'JSONURL',
     },
+    {
+      name: 'MyKeyName2:<bytes32>:<bool>',
+      key: '0x',
+      keyType: 'MappingWithGrouping',
+      valueType: 'bytes',
+      valueContent: 'VerifiableURL',
+    },
     {
       name: 'MyDynamicKey:<address>',
       key: '0x',
-      keyType: 'Singleton',
+      keyType: 'Mapping',
       valueType: 'bytes',
       valueContent: 'JSONURL',
     },
+    {
+      name: 'MyDynamicKey2:<address>',
+      key: '0x',
+      keyType: 'Mapping',
+      valueType: 'bytes',
+      valueContent: 'VerifiableURL',
```

Comment:
Indeed the standard states `URI`.

But let's triple check with @frozeman . See this PR from @JeneaVranceanu too: https://github.com/lukso-network/LIPs/pull/269#issuecomment-1854473600

Let's make sure we have the name correct and right in the first place, so that we don't have to re-release if it's incorrect.

Reply:
Yes please - one correct release 👉👈

Reply:
Confirming it's URI. Making changes now

Reply:
Cool. Yea the URL name came from JSONURL, sorry